docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -18,6 +18,7 @@ This aligns to the workflow shown in the [deployment architecture](../architectu
 It assumes the [Azure requirements](../requirements_data_azure.md#azure) are in place, including:
 
 * Azure subscription and service principal
+* If you want to provision infrastructure within a private network, then the Azure spoke virtual network and subnet for private endpoints must be provisioned for each environment. The hub network must contain a self-hosted agent. Please refer [Hub-Spoke network topology](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/hub-spoke?tabs=cli) for more details on Hub-spoke network topology in Azure.
 * Azure DevOps project with [Pipelines variable groups](../requirements_data_azure.md#azure-pipelines-variable-groups)
```

Comment:
Think you need to mention here that you need the network variable groups only if  you're provisioning in the private network (assuming that's right) - and also mention that on that page.

Reply:
That's correct its only required if we are provisioning in the private network,so we need to mention this at both pages or only in Variable group page?

Reply:
I'd mention on both

Reply:
Okay updated at both location.